servicex_app/servicex/templates/base.html

```diff
@@ -48,30 +48,39 @@
             {% endif %}
           </ul>
 
-          {% if config["ENABLE_AUTH"] %}
```

Comment:
Can you confirm that you have tested this both with Auth enabled and without? I could imagine there are some funny cases that could be triggered

Reply:
just tested it on River int xaod instance and ti looks fine.